### PR TITLE
Redesing - inspired by Lt-Mayonesa's material design

### PR DESCRIPTION
**Pull request by loro2**
Here is a simple redesign (#252) with support for switching between the new and original styles (in options, via css switching). The design work was basically done by Lt-Mayonesa in #254, but I implemented it in a more simplistic fashion, touching the html and javascript code as little as possible. The layout resembles the original, with the color scheme and some design cues taken from Lt-Mayonesa's work. I didn't use the mui framework, only vanilla css.

I think the whole #254 redesign can be implemented in this manner by building further upon this PR in the future.

### CODE REVIEW

File: js/row.js [hunk 1]
```diff
@@ -230,23 +231,24 @@ SQL.Row.prototype.load = function() { /* put data to expanded form */
 SQL.Row.prototype.redraw = function() {
 	var color = this.getColor();
 	this.dom.container.style.backgroundColor = color;
+	this.dom.container.style.borderColor = color;
```
Comment (loro2) [inline js/row.js]: This is the only functional change in this file. Sorry about the styling changes, I missed reverting them in this file.

**Comment by loro2** [comment]
Hello, I'd like to ping this PR. I understand this project is not high priority now, so this is just a friendly reminder of this PR if it got forgotten about unwillingly. The change to legacy code is quite trivial and there is a single new .css file. Thanks.

**Comment by ondras** [comment]
Hi,

ping received. I just returned from a 3-week vacation.

Please give me some time to review the code. I am totally willing to merge this, so checking the code is basically only a formality.

**Comment by ondras** [comment]
@loro2 , thanks a lot for your contribution!

**Pull request merged**